Update README and add in PR template (#1)
slack-samples/.github: README.md

# Overview

> This repository's purpose is to to configure org-wide Github templates for all repositories in the [`Slack Samples`](https://github.com/slack-samples)
> organization.
> 
> This repository is **public**, which is a required configuration so that all repositories in the org can access its templates and configurations.

Reviewer: ```suggestion
This repository's purpose is to configure GitHub templates for all repositories in the [`Slack Samples`](https://github.com/slack-samples)
organization.

This repository is **public**: a visibility requirement that enables all repositories in the organization to access shared templates and configurations.
```
